examples/hacker_news_assets/tox.ini

```diff
@@ -8,13 +8,14 @@ setenv =
 passenv = CI_* COVERALLS_REPO_TOKEN BUILDKITE SNOWFLAKE_ACCOUNT SNOWFLAKE_USER SNOWFLAKE_PASSWORD
 deps =
   -e ../../python_modules/dagster[mypy,test]
+  -e ../../python_modules/dagit
+  -e ../../python_modules/dagster-graphql
   -e ../../python_modules/libraries/dagster-pandas/
```

Comment:
Feel free to disregard, but I don't think we need dagster-pandas or dagster-spark anymore here.